### PR TITLE
LG-11407: Token page OIDC guide rework

### PR DESCRIPTION
**Pull request by nprimak**
this is part of the greater epic reworking the design and content of the developer docs 

Please refer to Figma and the OIDC Token page to compare to this design 

https://www.figma.com/file/3KSAr0qfDv8BXFIHyFEudZ/Login.gov-OIDC%2C-SAML%2C-and-UX-guides-(LG-10262%2C-LG-10514)?type=design&node-id=104-4195&mode=design&t=FIN6w52uL8lyaAdU-0

This page is fairly similar to the Authorization Page in terms of layout

### CODE REVIEW

**Review by karenstrother — COMMENTED** [review]
LGTM!

**Review by aduth — APPROVED** [review]
👍

File: _pages/oidc/token.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+- **sub** (string) — The subject, which must also be the `client_id`.
+- **aud** (string) — The audience, which should be (or, in the case of multiple audience values, include) the URL of the token endpoint, for example: `https://idp.int.identitysandbox.gov/api/openid_connect/token`
+- **jti** (string) — The JWT ID, a unique identifier for the token which can be used to prevent reuse of the token. Should be an unguessable, random string generated by the client.
+- **exp** (number) — The expiration time for this token. Should be an integer timestamp (number of seconds since the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time){:class="usa-link--external"}) and be a short period of time in the future (such as 5 minutes from now).
+{% endcapture %}
+
+
+<div class="grid-row grid-gap">
+  <div class="desktop:grid-col-8 mobile:grid-col-full">
+    <h2>Token</h2>
```
[the remark this answers is not in the document]
Comment (nprimak) [reply _pages/oidc/token.md:41]: I could try to create a "row' component kind of like the accordion component. The row gets reused a lot. Leaning toward that being an optimization in another ticket though, would that be okay? I do agree that repeating the `<code>` and `<a>` classes especially is annoying

File: _pages/oidc/token.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
+---
+title: OpenID Connect
+lead: >
+    [OpenID Connect](http://openid.net){:class="usa-link--external"} is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports [version 1.0](http://openid.net/specs/openid-connect-core-1_0.html){:class="usa-link--external"} of the specification and conforms to the [iGov Profile](https://openid.net/wg/igov){:class="usa-link--external"}.
+redirect_from:
+  - /openid-connect/
+sidenav:
```
[the remark this answers is not in the document]
Comment (nprimak) [reply _pages/oidc/token.md:7]: Yeah Zach suggested this too, I think we could pair this with moving things back into markdown as part of optimizing and making this more edit friendly like it was before.

File: _pages/oidc/token.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
+---
+title: OpenID Connect
+lead: >
+    [OpenID Connect](http://openid.net){:class="usa-link--external"} is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports [version 1.0](http://openid.net/specs/openid-connect-core-1_0.html){:class="usa-link--external"} of the specification and conforms to the [iGov Profile](https://openid.net/wg/igov){:class="usa-link--external"}.
+redirect_from:
+  - /openid-connect/
```
[the remark this answers is not in the document]
Comment (nprimak) [reply _pages/oidc/token.md]: that was copy pasted over from the original oidc page, I removed it thanks!

File: _pages/oidc/token.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="dev-doc-row">
+      <div class="grid-row">
+        <div class="grid-col-5">
+          <h4 class="parameters clearfix">code</h4>
+        </div>
+        <div class="grid-col-7 padding-bottom-2">
+            The authorization code returned by the <a class="usa-link" href="{{ site.baseurl }}/oidc/authorization/" class="usa-link">authorization response</a>.
```
[the remark this answers is not in the document]
Comment (nprimak) [reply _pages/oidc/token.md]: removed!

File: _pages/oidc/token.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nprimak) [reply _pages/oidc/token.md:39]: https://cm-jira.usa.gov/browse/LG-11556

**Review by anniehirshman-gsa — APPROVED** [review]
Looks great! Two tiny (not critical) visual design comments:

1. Just noticed that there's still a small 6px right padding on the second set of tabs in the code panel only (see other [PR comment](https://github.com/18F/identity-dev-docs/pull/375#issuecomment-1789367947))
2. Ideally it'd be great if there was a little more space above the h3 to visually separate new sections

Screen Shot 2023-11-07 at 3 31 33 PM

Thanks so much!

**Pull request merged**